src/algorithm/intersects.rs

```diff
@@ -442,6 +442,7 @@ mod test {
     #[test]
     fn point_intersects_line_test() {
         let p0 = Point::new(2., 4.);
+        let p1 = Point::new(0., 0.);
```

Comment:
No big deal, but rather than adding another point, it seems like the existing pattern of this test is to have one point, and then a line for every test case with a comment explaining why that particular line is interesting. 

As written, it's not clear why that point is an interesting test case outside of the context of this PR.

Maybe this would be a little clearer:

```
// Line below point
let line4 = Line::new(Point::new(0.,0.), Point::new(5., 0.)
assert!(!line4.intersects(&p0));

// Line above point
let line5 = Line::new(Point::new(0.,5.), Point::new(5., 5.)
assert!(!line5.intersects(&p0));
```